qiskit/assembler/assemble_schedules.py

```diff
@@ -92,6 +92,7 @@ def _assemble_experiments(
     instruction_converter = instruction_converter(qobj.PulseQobjInstruction,
                                                   **run_config.to_dict())
 
+    #ToDo(4872) remove the use of deprecated constructor Schedule(*schedulers)
```

Comment:
I marked all the calls to `Schedule` that use the deprecated argument.
Should we:
- Remove the use of the deprecated argument
- Leave the ToDo (possible updating the issue number with a newly created issue to remove said usage)
- Do nothing, remove all these comments.

Which option do you think is the best? Or would you like to handle these in yet a different way?